app/views/directives/pipeline-status.html

```diff
@@ -0,0 +1,34 @@
+<span class="pipeline-status" ng-class="status">
+  <span ng-switch="status" class="hide-ng-leave">
+    <!-- See https://github.com/jenkinsci/pipeline-stage-view-plugin/blob/master/rest-api/src/main/java/com/cloudbees/workflow/rest/external/StatusExt.java -->
+    <span ng-switch-when="ABORTED">
+      <i class="fa fa-ban text-muted" aria-hidden="true"></i>
+      <span class="sr-only">Aborted</span>
+    </span>
+    <span ng-switch-when="FAILED">
+      <span class="fa-stack">
+        <i class="fa fa-circle fa-stack-2x"></i>
+        <i class="fa fa-times fa-stack-1x fa-inverse"></i>
+      </span>
+      <span class="sr-only">Failed</span>
+    </span>
+    <span ng-switch-when="IN_PROGRESS">
+      <i class="fa fa-refresh fa-spin" aria-hidden="true"></i>
+      <span class="sr-only">In Progress</span>
+    </span>
+    <span ng-switch-when="NOT_EXECUTED">
+      <!-- No icon -->
+    </span>
+    <span ng-switch-when="PAUSED_PENDING_INPUT">
+      <i class="fa fa-pause" aria-hidden="true"></i>
+      <span class="sr-only">Paused Pending Input</span>
+    </span>
+    <span ng-switch-when="SUCCESS">
```

Comment:
should we have an ng-switch-default of a gray circle with a question mark or something?  i worry about things getting updated and new statuses appearing